apps/frontend/src/app/Database/DataEntry.ts

```diff
@@ -24,31 +24,45 @@ export class DataEntry<
     this.key = key
     this.init = init
     this.goKey = goKey
-    if (this.database.storage.keys.includes(key)) {
-      this.data = this.toCache(init(this.database))!
-      this.set(this.database.storage.get(key))
-    } else {
-      this.data = this.toCache(init(this.database))!
-      this.setCached(this.data)
-    }
+    this.set(
```

Comment:
I feel like this would read better as an if-statement, but that's personal choice. I just find this whole thing hard to read so that might be a me-problem